src/components/DocumentationTopic.vue

```diff
@@ -647,6 +676,47 @@ export default {
 <style scoped lang="scss">
 @import 'docc-render/styles/_core.scss';
 
+$space-size: 15px;
+
+.declaration-list-menu {
+  position: relative;
+  margin: 0 !important;
+  width: 100%;
+
+  .declaration-list-toggle {
+    display: flex;
+    flex-direction: row;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background: var(--color-fill); // cover border line
```

Comment:
If this is just meant to match the background color of the page, could we just set it to none or transparent?